### ISSUE TITLE
nami status doesn't return a non-zero exit code if a module isn't fully installed

### ISSUE THREAD

**Issue by prydonius** [bitnami/nami#24]
```
$ nami status ghost
nami    ERROR com.bitnami.ghost is not fully installed. You cannot execute commands
```

Returns exit code 0, later:

```
> nami status ghost
com.bitnami.ghost not running
```

returned non-zero exit code

**Comment by prydonius** [comment]
To add some context, I ran into this issue when trying to setup a readiness probe in the Ghost chart. Currently using `nami status ghost | grep "ghost is running"` as a workaround.

**Comment by juamedgod** [comment]
Fixed in #27 

The exit code is being now set to `5` to differentiate from a simply not running service. We plan to prepare a standard list of error codes for other failures but for now this will suffice.

The `1` exit code currently used to mark a non-running service will change in the near future so do not use it for equality comparisons. The recommended approach would be:

```
if (code == 0) {
  // running
} else if (code == 5) {
  // not initialized
} else {
  // not running
}
```

Or if you are fine with simply checking if the service is running or not:

```
if (code == 0) {
  // running
} else {
  // not running
}
```

But do not use:

```
if (code == 1) {
   // not running
}
```

**Comment by prydonius** [comment]
Thanks @juamedgod!!
